doc/uftrace-script.md

```diff
@@ -22,7 +22,8 @@ SCRIPT OPTIONS
 -S *SCRIPT_PATH*, \--script=*SCRIPT_PATH*
 :   Run a given script to do additional work at the entry and exit of function
     while processing recorded trace data.
-    The type of script is detected by the postfix such as '.py' for python.
+    The type of script is detected by the file extension.
+    for example '.py' for python and '.lua' for luajit.
```

Comment:
Please use a capital letter.
"for example" -> "For example,"

Reply:
Thanks, I fixed.